package.json

```diff
@@ -33,16 +33,16 @@
     "xmldom": "^0.5.0"
   },
   "scripts": {
-    "build": "tsc -p tsconfig.json && npm run build:browser",
-    "build:browser": "webpack --config browser-build.config.js",
-    "start:dev": "webpack --config dev-build.config.js --watch",
-    "prebuild": "npm run test",
-    "pretest": "npm run lint",
-    "prepublishOnly": "npm run build",
-    "test": "jest",
-    "test:watch": "jest --watchAll",
+    "build-browser": "webpack --config browser-build.config.js",
+    "build": "tsc -p tsconfig.json && npm run build-browser",
+    "lint:test:build": "npm run lint && npm run test && npm run build",
+    "lint:test": "npm run lint && npm run test",
     "lint": "eslint -c .eslintrc.js --ext .ts . && tsc --noEmit --project tsconfig.json",
-    "release": "np --no-yarn && git push https://github.com/geostyler/geostyler-sld-parser.git master --tags"
+    "prepublishOnly": "npm run lint:test:build",
+    "release": "np --no-yarn && git push https://github.com/geostyler/geostyler-sld-parser.git master --tags",
+    "start:dev": "webpack --config dev-build.config.js --watch",
+    "test-watch": "jest --watchAll",
```

Comment:
is `test-watch` insead of `test:watch` intentional?

Reply:
yes. I wanted to make clear that `:` calls multiple scripts in a row. I also renamed `start:dev` to `start-dev` with faf3b44 to be consistent.